web/src/features/panels/zone/MethodologyCard.tsx

```diff
@@ -3,16 +3,20 @@ import { Link } from 'components/Link';
 import { RoundedCard } from 'features/charts/RoundedCard';
 import { t } from 'i18next';
 import { EmapsIcon } from 'icons/emapsIcon';
+import { useState } from 'react';
 import trackEvent from 'utils/analytics';
 
 export default function MethodologyCard() {
+  const [accordionState, setAccordionState] = useState(true);
```

Comment:
The naming here was a bit confusing to me, I thought it would be open by default now - maybe worth renaming props to be consistent?

```suggestion
  const [isCollapsed, setIsCollapsed] = useState(true);
```